ckan/config/config_declaration.yaml

```diff
@@ -1715,3 +1715,15 @@ groups:
         type: int
         default: 180
         description: The option defines the timeout in seconds until giving up on a job
+
+
+  - annotation: NoneType Display Settings
+    options:
+      - key: ckan.null_label
```

Comment:
this is a datatables configuration option, would you move it to ckanext/datatables/config_declaration.yaml and update the name?

Reply:
moved to `datatablesview` plugin.